mlx/robot2rst.py

```diff
@@ -76,40 +75,30 @@ def _tweak_prefix(prefix):
 def main():
     '''Main entry point for script: parse arguments and execute'''
     parser = argparse.ArgumentParser(description='Convert robot to RsT.')
-    parser.add_argument("--robot", dest='robot_file', help='Input robot file', required=True,
+    parser.add_argument("-i", "--robot", dest='robot_file', help='Input robot file', required=True,
                         action='store')
-    parser.add_argument("--rst", dest='rst_file', help='Output RsT file', required=True,
+    parser.add_argument("-o", "--rst", dest='rst_file', help='Output RsT file', required=True,
                         action='store')
-    parser.add_argument("-k", dest='keyword_prefix', action='store', default='KEYWORD-',
-                        help="Overrides default 'KEYWORD-' prefix.")
-    parser.add_argument("-s", dest='setting_prefix', action='store', default='SETTING-',
-                        help="Overrides default 'SETTING-' prefix.")
     parser.add_argument("-t", dest='test_case_prefix', action='store', default='ITEST-',
                         help="Overrides default 'ITEST-' prefix.")
-    parser.add_argument("-v", dest='variable_prefix', action='store', default='VARIABLE-',
```

Comment:
I think we would also like to use ittem-attributes for variable prefixes (or itemized items like they were)

Reply:
I don't understand what you mean by this. Please elaborate further.

Reply:
This was a prefix for items that were robot variables and there above is prefix for settings and for keywords. I think we need to maintain/keep extracting those, to get detailed enough rst report summary.

Reply:
We decided to stop extracting them and only extract test cases with their tags and documentation